src/ngrest/base/Controller.php

```diff
@@ -77,6 +77,13 @@ class Controller extends \luya\admin\base\Controller
      * + ng-href: 'katalogadmin/produkt/xml-download' (an example if you like to use yii response sendContentAsFile)
      */
     public $globalButtons = [];
+
+    /**
+     * @var string|array|\luya\admin\ngrest\render\RenderCrudInterface
+     *
```

Comment:
Please add also an example of people could use this like `If you need to override view files you can create class which implements the RenderCrudInterface and configure the object ``` [' class' => 'xyz', 'foo' => 'bar']``` `